### PR TITLE
Forcing the breadcrumb li to render a greater than sign instead of a slash.

### LINKED ISSUE #211: Use ">" for breadcrumbs instead of "/"

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Fixes #211 by overriding what Bootstrap automatically adds as a breadcrumb separator.

### CODE REVIEW

**Comment by EdwinGuzman** [comment]
Updated the PR to also fix #185 which is closely related.

**Comment by aslagle** [comment]
@EdwinGuzman this looks great. There is one place where the last breadcrumb maybe should be a link though - when you're on the book details page, it's useful to be able to go back to the page you came from.

**Comment by EdwinGuzman** [comment]
Yeah, I just noticed that as well. If we follow this pattern, then it'd be good to have a "back" button on the book details page instead of updating the breadcrumb for this use case. Otherwise, I'd imagine that it'll be tricky to let the breadcrumb know that it's on the book details page?

**Comment by aslagle** [comment]
I think you could pass it a prop from Root to say whether to make the last one a link. Root renders the book so it knows when you're on that page.

**Comment by EdwinGuzman** [comment]
Updated and got tests working, too.

File: packages/opds-web-client/src/components/__tests__/Breadcrumbs-test.tsx [hunk 1]
```diff
@@ -30,12 +30,29 @@ describe("Breadcrumbs", () => {
...
+    // The current page is no longer a link unless `currentLink` is true:
+    expect(links.length).to.equal(1);
     expect(links.at(0).props().children).to.contain("2nd title");
     expect(links.at(0).props().collectionUrl).to.equal("2nd url");
     // last link is wrapped in <strong>
-    expect((links.at(1).props().children as any).props.children).to.contain("last title");
+    expect(currentLink.text()).to.equal("last title");
+  });
+
+  it("shows links with bootstrap classes", () => {
```
Comment (aslagle) [inline packages/opds-web-client/src/components/__tests__/Breadcrumbs-test.tsx]: I would rename these two tests to reflect that they're testing with and without the current link now.

**Comment by aslagle** [comment]
Works great, this is good to merge if you update those test names. Could you also change the version number? And tomorrow I'll add you to the npm package so you can publish :)

**Pull request merged**